EdvClient.js

```diff
@@ -650,8 +656,39 @@ export class EdvClient {
     if(!(config.controller && typeof config.controller === 'string')) {
       throw new TypeError('"config.controller" must be a string.');
     }
-    const response = await axios.post(
-      url, config, {headers: DEFAULT_HEADERS, httpsAgent});
+
+    // no invocationSigner was provided, submit the request without a zCap
+    if(!invocationSigner) {
+      const response = await axios.post(
+        url, config, {headers: DEFAULT_HEADERS, httpsAgent});
+      return response.data;
+    }
+
+    _assertInvocationSigner(invocationSigner);
+
+    if(!capability) {
+      if(url.indexOf(':')) {
+        capability = `${url}/zcaps/configs`;
+      // eslint-disable-next-line no-undef
+      } else if(self) {
```

Comment:
This does seem a bit weird, but if it works for the use case now, no problem.   Maybe in the future using something with `globalThis` would be nicer.